libass/C/be_blur.c

```diff
@@ -24,69 +24,51 @@
 #include <stdint.h>
 
 
+static inline uint16_t sum_swap(uint16_t *ptr, uint16_t val)
+{
+    uint16_t sum = *ptr + val;
+    *ptr = val;
+    return sum;
+}
+
```

Comment:
I dislike both `sum_swap` and `sum_copy`. Maybe something with intent and not description, like `running_sum` or `pair_sum`, would be better?

Reply:
`running_sum` seems good to me.

Reply:
Sum of sequence til some point is usually called running sum and it's not a case here. I think
```c
static inline uint16_t pair_sum(uint16_t *prev, uint16_t next)
```
is better. Or maybe `cur` or `val` instead of `next`.

Reply:
`pair_sum` reads like just the sum of two numbers to me; while the function does return the sum of two numbers, it also sets the value of the first arg to the second arg's. How about `sum_history` or `sum_memory` as it records/memorises the last summand? Or `sum_moving` because the value of the pointer argument moves through all used summand?

Reply:
`sliding_sum`? `windowed_sum`?